socorro/external/rabbitmq/test_rabbitmq.py

```diff
@@ -1,6 +1,6 @@
 import pika
```

Comment:
I paid very little attention to this app except for it's ability to purge a rabbit queue.  I moved it and stopped it from spewing to stdout.   Looking at the code, it the only useful thing that it does is purge queue.  It's not used anywhere else in the system.

Query the original author, it was a utility to assist in the setup of rabbitmq.  It was for ad hoc use to try different settings in the process of originally setting up RMQ.  The only reason it still exists is because of the ability to purge queues with it.  It is a utility to manipulate RMQ that only persists to help the integration tests.

I think I'm going to refactor it out of existence...